src/curves/bip32Ed25519.ts

```diff
@@ -0,0 +1,94 @@
+import { stringToBytes } from '@metamask/utils';
+import { ed25519 } from '@noble/curves/ed25519';
+import { pbkdf2 } from '@noble/hashes/pbkdf2';
+import { sha512 } from '@noble/hashes/sha512';
+import libsodium from 'libsodium-wrappers-sumo';
+
+export const curve = ed25519.CURVE;
+
+/**
+ * Named after whitepaper: BIP32-Ed25519 Hierarchical Deterministic Keys over a Non-linear Keyspace
+ * https://input-output-hk.github.io/adrestia/static/Ed25519_BIP.pdf
+ */
+export const name = 'bip32Ed25519';
```

Comment:
I'd consider adding "bip32" as a suffix as I'd consider it "semantically" less important than the curve itself, and I'd separate it with an underscore as mixing two abbreviations with camel case looks a bit odd to me

```suggestion
export const name = 'ed25519_bip32';
```